tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -1828,7 +1881,7 @@ def _handle_canceled_requests(self):
             self.canceled_req_ids.clear()
 
     @nvtx_range("_enqueue_responses")
-    def _enqueue_responses(self, responses: Dict[int, LlmResponse]):
+    def _enqueue_responses(self, responses: List[Tuple[int, LlmResponse]]):
```

Ruff (0.12.2)

Undefined name `Tuple`

(F821)